packages/editor/src/components/post-excerpt/panel.js

```diff
@@ -79,6 +79,12 @@ function ExcerptPanel() {
 	);
 }
 
+/**
+ * `PostExcerptPanel` is a wrapper component that uses the `PostExcerptCheck`
```

Comment:
Maybe something like:

> Is rendered if the post type supports excerpts and allows editing the excerpt.

We can omit the name at the start of the docs, since it's going to be a bit duplicative in the docs..